lib/robots/dor_repo/was_crawl_dissemination/cdxj_generator.rb

```diff
@@ -8,11 +8,7 @@ def initialize
           super('wasCrawlDisseminationWF', 'cdxj-generator')
         end
 
-        # `perform` is the main entry point for the robot. This is where
```

Comment:
curiously - this context almost helps me, while the  `super()` line in constructor with line comment feels superfluous.  But I can live without this comment.   

Maybe someday we can rid ourselves of these centralized/decentralized robots.